website/docs/docs/cloud/connect-data-platform/connnect-bigquery.md

```diff
@@ -52,6 +52,122 @@ As an end user, if your organization has set up BigQuery OAuth, you can link a p
 
 To learn how to optimize performance with data platform-specific configurations in dbt Cloud, refer to [BigQuery-specific configuration](/reference/resource-configs/bigquery-configs).
 
+### Optional configurations
+
+In BigQuery, optional configurations let you tailor settings for tasks such as query priority, dataset location, job timeout, and more. These options give you greater control over how BigQuery functions behind the scenes to meet your requirements.
+
+To customize your optional configurations in dbt Cloud:
+
+- Click your name at the bottom left-hand side bar menu in dbt Cloud
```

Comment:
two things:
- since these are steps, they should be numbered [per our style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#lists)

- these steps are a little confusing -- i don't see any 'settings' option when i click on the bq project. i think you had it right the first time using 'account home' or account name' as the first step:
1. Click **Account home** from the left-hand sidebar menu in dbt Cloud and then **Account settings**.
2. From there, click **Project** and select your BigQuery project
3. Select the **BigQuery** under **Development Connection**. 
4. Click **Edit** and then scroll down to **Optional settings**.

i feel this is more direct and shortens the steps to 4 instead of 7.